### ISSUE TITLE
Error when trying to start new run with latest unshackle and skin editor.

### ISSUE THREAD

**Issue by SkyeOfBreeze** [EvaisaDev/evaisa.skineditor#1]
most times noita does not seem to output anything in the log, but this time it spit something out. hope it helps. Unshackle launches fine on its own

```
Noita (Aug 12 2024) - 20241202-120848
-------------------------------------------
Lua error - [string "mods/evaisa.unshackle/lib/input.lua"]:206: wrong number of type parameters at line 85
  Stack traceback:
	  [string "mods/evaisa.unshackle/lib/input.lua"]:206: in function <[string "mods/evaisa.unshackle/lib/input.lua"]:123>
	  [C]: in function 'cdef'
	  [string "mods/evaisa.unshackle/lib/ffi_extensions.lua"]:97: in function 'f'
	  [string "__loaded={} dofile=function(filename) local f..."]:1: in function 'dofile'
	  [string "mods/evaisa.unshackle/init.lua"]:67: in main chunk
Lua (DoFile) error at 'mods/evaisa.unshackle/init.lua': [string "mods/evaisa.unshackle/lib/input.lua"]:206: wrong number of type parameters at line 85
  Stack traceback:
	  [string "mods/evaisa.unshackle/lib/input.lua"]:206: in function <[string "mods/evaisa.unshackle/lib/input.lua"]:123>
	  [C]: in function 'cdef'
	  [string "mods/evaisa.unshackle/lib/ffi_extensions.lua"]:97: in function 'f'
	  [string "__loaded={} dofile=function(filename) local f..."]:1: in function 'dofile'
	  [string "mods/evaisa.unshackle/init.lua"]:67: in main chunk
```

**Comment by EvaisaDev** [comment]
> most times noita does not seem to output anything in the log, but this time it spit something out. hope it helps. Unshackle launches fine on its own
> 
> ```
> Noita (Aug 12 2024) - 20241202-120848
> -------------------------------------------
> Lua error - [string "mods/evaisa.unshackle/lib/input.lua"]:206: wrong number of type parameters at line 85
>   Stack traceback:
> 	  [string "mods/evaisa.unshackle/lib/input.lua"]:206: in function <[string "mods/evaisa.unshackle/lib/input.lua"]:123>
> 	  [C]: in function 'cdef'
> 	  [string "mods/evaisa.unshackle/lib/ffi_extensions.lua"]:97: in function 'f'
> 	  [string "__loaded={} dofile=function(filename) local f..."]:1: in function 'dofile'
> 	  [string "mods/evaisa.unshackle/init.lua"]:67: in main chunk
> Lua (DoFile) error at 'mods/evaisa.unshackle/init.lua': [string "mods/evaisa.unshackle/lib/input.lua"]:206: wrong number of type parameters at line 85
>   Stack traceback:
> 	  [string "mods/evaisa.unshackle/lib/input.lua"]:206: in function <[string "mods/evaisa.unshackle/lib/input.lua"]:123>
> 	  [C]: in function 'cdef'
> 	  [string "mods/evaisa.unshackle/lib/ffi_extensions.lua"]:97: in function 'f'
> 	  [string "__loaded={} dofile=function(filename) local f..."]:1: in function 'dofile'
> 	  [string "mods/evaisa.unshackle/init.lua"]:67: in main chunk
> ```

Spent a few days trying to wrap my head around this issue and i think it should be solved now.
Spoiled alert: It was a multithreading issue.
For some reason in my own save file, single threaded loading was turned on, which prevented my game from crashing.
I have no idea why this was turned on, but it caused so much fucking confusion because i could not reproduce the crash until i figured that out.
TLDR: Multi threading fucking sucks

**Comment by SkyeOfBreeze** [comment]
ig I should have updated here afterwards, but just now tested it and it seems to be working 🙂
